okhttp/third_party/okhttp/main/java/io/grpc/okhttp/internal/framed/Hpack.java

```diff
@@ -57,69 +57,85 @@
-      new io.grpc.okhttp.internal.framed.Header("access-control-allow-origin", ""),
-      new io.grpc.okhttp.internal.framed.Header("age", ""),
-      new io.grpc.okhttp.internal.framed.Header("allow", ""),
-      new io.grpc.okhttp.internal.framed.Header("authorization", ""),
-      new io.grpc.okhttp.internal.framed.Header("cache-control", ""),
-      new io.grpc.okhttp.internal.framed.Header("content-disposition", ""),
-      new io.grpc.okhttp.internal.framed.Header("content-encoding", ""),
-      new io.grpc.okhttp.internal.framed.Header("content-language", ""),
-      new io.grpc.okhttp.internal.framed.Header("content-length", ""),
-      new io.grpc.okhttp.internal.framed.Header("content-location", ""),
-      new io.grpc.okhttp.internal.framed.Header("content-range", ""),
-      new io.grpc.okhttp.internal.framed.Header("content-type", ""),
-      new io.grpc.okhttp.internal.framed.Header("cookie", ""),
-      new io.grpc.okhttp.internal.framed.Header("date", ""),
-      new io.grpc.okhttp.internal.framed.Header("etag", ""),
-      new io.grpc.okhttp.internal.framed.Header("expect", ""),
-      new io.grpc.okhttp.internal.framed.Header("expires", ""),
-      new io.grpc.okhttp.internal.framed.Header("from", ""),
-      new io.grpc.okhttp.internal.framed.Header("host", ""),
-      new io.grpc.okhttp.internal.framed.Header("if-match", ""),
-      new io.grpc.okhttp.internal.framed.Header("if-modified-since", ""),
-      new io.grpc.okhttp.internal.framed.Header("if-none-match", ""),
-      new io.grpc.okhttp.internal.framed.Header("if-range", ""),
-      new io.grpc.okhttp.internal.framed.Header("if-unmodified-since", ""),
-      new io.grpc.okhttp.internal.framed.Header("last-modified", ""),
-      new io.grpc.okhttp.internal.framed.Header("link", ""),
-      new io.grpc.okhttp.internal.framed.Header("location", ""),
-      new io.grpc.okhttp.internal.framed.Header("max-forwards", ""),
-      new io.grpc.okhttp.internal.framed.Header("proxy-authenticate", ""),
-      new io.grpc.okhttp.internal.framed.Header("proxy-authorization", ""),
-      new io.grpc.okhttp.internal.framed.Header("range", ""),
-      new io.grpc.okhttp.internal.framed.Header("referer", ""),
-      new io.grpc.okhttp.internal.framed.Header("refresh", ""),
-      new io.grpc.okhttp.internal.framed.Header("retry-after", ""),
-      new io.grpc.okhttp.internal.framed.Header("server", ""),
-      new io.grpc.okhttp.internal.framed.Header("set-cookie", ""),
-      new io.grpc.okhttp.internal.framed.Header("strict-transport-security", ""),
-      new io.grpc.okhttp.internal.framed.Header("transfer-encoding", ""),
-      new io.grpc.okhttp.internal.framed.Header("user-agent", ""),
-      new io.grpc.okhttp.internal.framed.Header("vary", ""),
-      new io.grpc.okhttp.internal.framed.Header("via", ""),
-      new io.grpc.okhttp.internal.framed.Header("www-authenticate", "")
-  };
+  private static final io.grpc.okhttp.internal.framed.Header[] STATIC_HEADER_TABLE =
```

Comment:
Please do not make excessive changes.

Reply:
I am not changing any code, just the style. I don't think this worth another pr.

Reply:
No, fix this PR. Do not modify anything other than the `final`.